content/customize-gnome.md

```diff
@@ -18,117 +18,210 @@ section: software
 
-To install a customization, simply head to [extensions.gnome.org](https://extensions.gnome.org) to install, remove, and configure currently installed customizations.  A web browser plugin is needed to interact with the site, and it will prompt for installation if needed.  Remember to refresh the page after the plugin is installed.
-
-We will share some of the more popular extensions, but there are many more to choose from!  If one is found and it isn't currently working, or breaks with an update, then try a different extension to see if it works better.
+**NOTE:** There are many extensions to choose from! If one isn't currently working, or breaks with an update, then try a different extension to see if it works better.
 
 ---
 
-## User Themes
+## Included Extensions
 
-[User Themes](https://extensions.gnome.org/extension/19/user-themes/)
+<details>
+  <summary>Click to expand</summary>
 
-This extension allows for the desktop to be themed in further ways than default.  Many themes and extensions can be installed from [opendesktop.org](https://www.opendesktop.org/s/Gnome), among others.  This extension is installed by default in Pop!_OS, and can be added to Ubuntu 18.04.
+Pop!\_OS includes the GNOME ```Extensions``` app by default. Which lists both system-installed, and user-installed extensions.
 
----
+![extensions-app-window](/images/customize-gnome/extensions-app-window.png)
 
-## Dash to Dock
+Each extension has options for customizing settings (if supported in Extension metadata), or viewing the webpage for the extension to learn more information. The Extensions app also allows you to quickly enable and disable installed extensions.
 
-[Dash to Dock](https://extensions.gnome.org/extension/307/dash-to-dock/)
+Pop!\_OS includes the following extensions by default. In most cases, these extensions should be left **enabled**.
 
-![dash-to-dock](/images/customize-gnome/dash-to-dock.png)
+### [Cosmic Dock](https://github.com/pop-os/cosmic-dock)
 
-This extension brings a launcher similar to the Unity launcher on the left side, or the macOS launcher on the bottom.  It is highly customizable and allows for different locations, styles, sizes, and layouts.  This is included by default in Ubuntu 18.04 and can be added to Pop!_OS.
+<details>
+  <summary>Click to expand</summary>
 
----
+A fork of Dash to Dock customized for Pop!\_OS.
 
-## Backslide
+![cosmic-dock-close](/images/customize-gnome/cosmic-dock-close.png)
 
-[Backslide](https://extensions.gnome.org/extension/543/backslide/)
+**NOTE:** It is **not** recommended to have both Cosmic Dock and Dash-to-Dock installed an enabled at the same time. User Interface (UI) bugs will occur. Choose one or the other.
```

Comment:
See above comment regarding the quotation designator

Reply:
See my above reply. :) I'll edit to make it consistent.

Reply:
Fixed.